source/PasswordManager.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+
+#include "../headers/PasswordManager.h"
+
+PasswordManager::PasswordManager() {}
```

Comment:
We don't use this constructor but if we really want preserve an option to default-construct PasswordManager class, putting `PasswordManager() = default` in the header will do just that.